doc/doc-site/docs/overview/orders.md

```diff
@@ -215,297 +215,214 @@ Econia does not use a critical height of 1. The actual critical height is define
->>> tick_size_integer
-1
+>>> lot_size = "0.00001"
+>>> tick_size = "0.00001" # changed by / 1000
+>>> get_min_quote_per_base_nominal(lot_size, tick_size)
+1.0
+>>> get_max_quote_per_base_nominal(lot_size, tick_size)
+4294967295.0
 ```
 
-However, the new decimal order size granularity corresponds to 1.779227 `USDC` nominal, which may not be considered granular enough.
-Hence alternative parametric inputs might entail:
+That's a minimum price (and price granularity) of \$1, as well as a maximum price that's "plenty" high.
+This could work; if a price granularity of one penny is desired, then lower the tick size by a further factor of 100:
 
-1. 0.00005 `wBTC` decimal order size granularity (corresponding to 0.8896135 `USDC` nominal).
-1. 0.02 `USDC` decimal price granularity.
+```python
+>>> lot_size = "0.00001"
+>>> tick_size = "0.0000001" # changed by / 1000 / 100
+>>> get_min_quote_per_base_nominal(lot_size, tick_size)
+0.01
+>>> get_max_quote_per_base_nominal(lot_size, tick_size)
+42949672.95
+```
 
-Here, the total quote amount increment again corresponds to an integer multiple of `USDC` subunits, for a tick size of 1:
+**...but wait a second!** `USDC` *has 6 decimals and above, we've used 7*.
+We can see that this tick size is problematic by using `get_tick_size_integer` from the SDK:
 
-```python
->>> lot_size_decimal = dec('0.00005')
->>> tick_size_decimal = dec('0.02')
->>> to_check = (lot_size_decimal * tick_size_decimal
-...             * 10 ** decimals_quote).normalize()
->>> tick_size_integer = int(to_check)
->>> assert to_check >= 1 and to_check == tick_size_integer
->>> tick_size_integer
-1
 ```
+>>> get_tick_size_integer("0.0000001", 6)
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+  File "/Users/egd/projects/econia/src/python/sdk/econia_sdk/utils/decimals.py", line 46, in get_tick_size_integer
+    return get_lot_size_integer(smallest_decimal_unit, coin_decimals)
+  File "/Users/egd/projects/econia/src/python/sdk/econia_sdk/utils/decimals.py", line 31, in get_lot_size_integer
+    raise ValueError("Decimal unit too small to represent with 1 subunit")
+ValueError: Decimal unit too small to represent with 1 subunit
+```
+
+Thus, holding lot size constant, the most price granularity possible is \$0.10 which corresponds to a tick size of 0.000001 `USDC` (the minimum possible) and lot size of 0.00001 `WBTC`.
+
+If we increase lot size by a factor of 10 to 0.0001 `WBTC` which corresponds to an order value granularity of \$1.779 instead of \$0.1779, and keep the above tick size, then we have a price granularity of one penny:
 
-Note, however, that compared with the other viable parameter set, this set trades off size precision for price precision:
-twice as much size granularity entails one half as much price granularity.
+```python
+>>> lot_size = "0.0001" # changed by * 10
+>>> tick_size = "0.000001"
+>>> get_min_quote_per_base_nominal(lot_size, tick_size)
+0.01
+```
 
```

Comment:
The below note is out of date due to due to changes in market order APIs that never got documented here, and it is probably not needed. Will you please take it out?